README.md

```diff
@@ -39,6 +39,7 @@ launching a Kubernetes cluster hosted on AWS.
 * Ability to generate configuration files for AWS [CloudFormation](https://aws.amazon.com/cloudformation/) and Terraform [Terraform configuration](/docs/terraform.md)
 * Supports custom Kubernetes [add-ons](/docs/addons.md)
 * Command line [autocompletion](/docs/cli/kops_completion.md)
+* Manifest Based API [Configuration](/docs/manifests_and_customizing_via_api.md)
```

Comment:
It is a term from kubernetes https://www.google.com/search?q=kubernetes+manifest&rlz=1C5CHFA_enUS691US691&oq=kubernetes+manifest&aqs=chrome..69i57j0j69i60l3j0.4401j0j7&sourceid=chrome&ie=UTF-8